### DISCUSSION TITLE
Snapshots Only Accepting Default Path

### DISCUSSION THREAD

**Discussion by MOHAA2002** [stella-emu/stella#1006] [Q&A] [answered]
Taking snapshots on the default path "~\Desktop\" results in screenshots showing up, but changing it to a different directory is the complete opposite.

**Comment by thrust26** [comment]
What do you mean? What happens to the snapshots? And which OS are you using? 

I am saving my snapshots in a subfolder of the user's document folder without any problems (Windows 10).

**Comment by MOHAA2002** [comment]
> What do you mean? What happens to the snapshots? And which OS are you using?
> 
> I am saving my snapshots in a subfolder of the user's document folder without any problems (Windows 10).

On Windows 11, the snapshots get saved but don't appear on Stella's main menu.

**Comment by thrust26** [comment, marked as answer]
There are two settings, one in Options/Snapshots and one in Options/User Interface/Launcher. If you want your snapshots to appear in the launcher, they must be identical.
